packages/gui/src/components/walletConnect/WalletConnectAddConnectionDialog.tsx

```diff
@@ -212,6 +221,23 @@ export default function WalletConnectAddConnectionDialog(props: WalletConnectAdd
     return null;
   }
 
+  function renderQuietModeOption() {
+    return (
+      <Flex
+        sx={{ cursor: 'pointer' }}
+        alignItems="center"
+        onClick={() => {
+          toggleBypassCheckbox(!bypassCheckbox);
+        }}
+      >
+        <Checkbox checked={bypassCheckbox} disableRipple sx={{ paddingLeft: 0 }} />
+        <Typography variant="body2">
+          <Trans>Bypass confirmation for all read-only commands</Trans>
+        </Typography>
```

Comment:
Tweak wording and add an informative tooltip on hover.

⚠️ NOTE: `TooltipIcon` needs to be imported from `@chia-network/core` ⚠️

```suggestion
        <Flex flexDirection="row" alignItems="center" gap={1}>
          <Typography variant="body2">
            <Trans>Skip confirmation for all read-only commands</Trans>
          </Typography>
          <TooltipIcon>
            <Trans>
              By default, a prompt will be presented each time a WalletConnect command is invoked. Select this option if
              you would like to skip the prompt for all read-only WalletConnect commands.
              <p />
              Commands that create a transaction or otherwise modify your wallet will still require confirmation.
            </Trans>
          </TooltipIcon>
        </Flex>
      </Flex>
```